src/app/main/listing/listing-form/listing-form.component.html

```diff
@@ -60,8 +60,8 @@
         />
       </mat-form-field>
 
-      <mat-error data-input-image-url-error *ngIf="formGroupErrors?.imageUrl?.required">
-        Image url is required
+      <mat-error>
+        {{formGroupErrors?.imageUrl | formatError: 'Image Path'}}
```

Comment:
the argument is optional. when provided, it is displayed

Reply:
only one error at a time is shown based on the order of the FormControl's validators array